### DISCUSSION TITLE
[QA] exception in phase 'semantic analysis' in source unit 'Script1.groovy' Unsupported class file major version 61

### DISCUSSION THREAD

**Discussion by liming1010** [apache/incubator-hugegraph#2357] [Q&A] [answered]
### Bug Type (问题类型)

server status (启动/运行异常)

### Before submit

- [X] 我已经确认现有的 [Issues](https://github.com/apache/hugegraph/issues) 与 [FAQ](https://hugegraph.apache.org/docs/guides/faq/) 中没有相同 / 重复问题 (I have confirmed and searched that there are no similar problems in the historical issue and documents)

### Environment (环境信息)

- Server Version: 1.0.0 (Apache Release Version)

### Expected & Actual behavior (期望与实际表现)

运行gremlin-console.sh 脚本出现错误：
        at org.codehaus.groovy.control.CompilationUnit.applyToSourceUnits(CompilationUnit.java:969)

### Vertex/Edge example (问题点 / 边数据举例)

```javascript
无
```

### Schema [VertexLabel, EdgeLabel, IndexLabel] (元数据结构)

```javascript
无
```

**Comment by imbajin** [comment, marked as answer]
maybe could search `Unsupported class file major version 61` directly, seems u use the **mismatch Java version(env)**?

and u could try server's [docker images](https://hub.docker.com/r/hugegraph/hugegraph) to avoid the env problem, then find the difference (if the docker images `1.0.0` have the same problem but the **latest** image doesn't, could refer the PR[1] below)
 
refer:
- #2231 (use the master code / latest docker images)
- groovy 3.x need  Java8+
- groovy 4.x need Java11+
- we will prepare to release V1.2.0 in this month

**Comment by liming1010** [comment]
ok , is woking
env of jdk21, need jdk11
How long will jdk21 be supported

**Reply by imbajin** [reply]
Thanks for the feedback, the support of Java21 depends on **backend storage** and updates to dependencies such as **tinkerpop**, and hugegraph is actually unimpeded 🔢  (we will support `Java17` first)

BTW, if this problem solve, click the `Mark as answer` button to finish it
